### PR TITLE
Correct the hybrid manifest so a hybrid network deploys

### PR DESCRIPTION
**Pull request by JacobTanenbaum**
Originally I defined the wrong object in the manifests I defined a network config instead of an operator config. This corrects the creation of the cluster-network operator and a hybrid capable network will deploy

### CODE REVIEW

File: ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml [hunk 1]
```diff
@@ -31,3 +31,7 @@ tests:
   steps:
     cluster_profile: packet
     workflow: baremetalds-e2e
+- as: e2e-ovn-hybrid-step-registry
+  steps:
+    cluster_profile: aws
+    workflow: origin-e2e-aws-ovn-hybrid
```
Comment (aravindhp) [inline ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml:37]: Does the test that is part of this workflow check if OVN hybrid overlay is enabled?

Comment (JacobTanenbaum) [reply ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml:37]: no, we have no testing around the hybrid overlay

**Comment by JacobTanenbaum** [comment]
from a test run: [output logs from gather extra](https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_release/8657/rehearse-8657-pull-ci-openshift-ovn-kubernetes-master-e2e-ovn-hybrid-step-registry/2/artifacts/e2e-ovn-hybrid-step-registry/gather-extra/pods/openshift-ovn-kubernetes_ovnkube-master-kvqbn_ovnkube-master.log)

the line we are interested in:
`+ exec /usr/bin/ovnkube --init-master ip-10-0-135-230.us-west-2.compute.internal --config-file=/run/ovnkube-config/ovnkube.conf --ovn-empty-lb-events --loglevel 4 --enable-hybrid-overlay --no-hostsubnet-nodes=kubernetes.io/os=windows --hybrid-overlay-cluster-subnets=10.132.0.0/14 --metrics-bind-address 0.0.0.0:9102 --sb-address ssl:10.0.135.230:9642,ssl:10.0.142.9:9642,ssl:10.0.151.157:9642 --sb-client-privkey /ovn-cert/tls.key --sb-client-cert /ovn-cert/tls.crt --sb-client-cacert /ovn-ca/ca-bundle.crt --nb-address ssl:10.0.135.230:9641,ssl:10.0.142.9:9641,ssl:10.0.151.157:9641 --nb-client-privkey /ovn-cert/tls.key --nb-client-cert /ovn-cert/tls.crt --nb-client-cacert /ovn-ca/ca-bundle.crt --nbctl-daemon-mode true`

the ovnkube-master process is started with the hybrid overlay configuration

File: ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml (hunk 1 above)
Comment (ravisantoshgudimetla) [reply ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml:37]: Please change the name to something else as we're not running e2e's with ovn-hybrid enabled

Comment (aravindhp) [reply ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml:37]: @ravisantoshgudimetla that is exactly what @JacobTanenbaum is doing except there is no e2e that tests if ovn-hybrid got enabled.

Comment (ravisantoshgudimetla) [reply ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml:37]: Right that's what we discussed on slack, so I am fine with the change.

**Pull request merged**